### DISCUSSION TITLE
Golang functions initialSubscriptionPosition not supported?

### DISCUSSION THREAD

**Discussion by conor-nsurely** [apache/pulsar#22445] [Q&A]
The streamnative docs https://docs.streamnative.io/docs/function-develop-golang#feature-matrix

Say that this feature is not supported but I can't see anything about it in the apache pulsar documentation.
Would anyone be able to clarify this for me please?

**Comment by lhotari** [comment]
I don't have an answer, but this seems to be related to https://github.com/apache/pulsar/issues/22349 and https://github.com/apache/pulsar/pull/22350. Perhaps @jiangpengcheng could explain?

**Comment by jiangpengcheng** [comment]
the Go instance still lacks some configuration, I will create a new PR to add them
